feconf.py

```diff
@@ -68,6 +69,9 @@
 RULES_DESCRIPTIONS_FILE_PATH = os.path.join(
     os.getcwd(), 'extensions', 'interactions', 'rules.json')
 
+# The ID of the classifier to be used as a default string classifier.
+DEFAULT_STRING_CLASSIFIER = 'LDAStringClassifier'
```

Comment:
Oh, I think I misunderstood the point of this constant. DEFAULT_STRING_CLASSIFIER_ID should work.